nixos/modules/services/networking/syncthing.nix

```diff
@@ -213,7 +213,7 @@ in {
 
             path = mkOption {
               type = types.str;
-              default = name;
+              default = "~/${name}";
```

Comment:
I wonder if the folders should rather be placed in `services.syncthing.dataDir`? From the description, I'm surprised it's not already the case:

> The path where synchronised directories will exist.

Reply:
Tested this locally, it does try to create `/name`. Weird.

Reply:
Ah, duh, they're just relative to the current working directory, which is `/` by default. Maybe we should add `WorkingDirectory = "~";`?

Reply:
> I wonder if the folders should rather be placed in `services.syncthing.dataDir`?

That's exactly what `~` refers to afaik.

Reply:
> Ah, duh, they're just relative to the current working directory, which is `/` by default. Maybe we should add `WorkingDirectory = "~";`? That would solve the problem without needing to mess with the default `path`.

Sounds like the cleanest solution.

Reply:
Actually, `~` is only `dataDir` by default if using the default `syncthing` user. In general they may be different.

So I amend my proposition: we should set `WorkingDirectory = cfg.dataDir;`.